problem_builder/questionnaire.py

```diff
@@ -158,7 +158,14 @@ def author_edit_view(self, context):
         Add some HTML to the author view that allows authors to add choices and tips.
         """
         fragment = self.get_author_edit_view_fragment(context)
-        fragment.add_content(loader.render_template('templates/html/questionnaire_add_buttons.html', {}))
+
+        # Let the parent block determine whether to display review-related questionnaire options.
```

Comment:
@pomegranited Nit: "review-related questionnaire options" is a bit vague, maybe change it to "buttons to add review-related child blocks"?